environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -0,0 +1,369 @@
+          image: ${IMAGE}
+          name: thanos-receive
+          ports:
+          - containerPort: 10901
+            name: grpc
+          - containerPort: 19291
+            name: remote-write
+          volumeMounts:
+          - mountPath: /var/thanos/tsdb
+            name: data
+            readOnly: false
+        volumes:
+        - emptyDir: {}
+          name: data
+    volumeClaimTemplates: []
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app: thanos-store
+    name: thanos-store
+    namespace: ${NAMESPACE}
+  spec:
+    clusterIP: None
+    ports:
+    - name: grpc
+      port: 10901
+      targetPort: 10901
+    selector:
+      app: thanos-store
+- apiVersion: apps/v1
+  kind: StatefulSet
+  metadata:
+    labels:
+      app: thanos-store
+    name: thanos-store
+    namespace: ${NAMESPACE}
+  spec:
+    replicas: ${{THANOS_STORE_REPLICAS}}
+    selector:
+      matchLabels:
+        app: thanos-store
+    serviceName: thanos-store
+    template:
+      metadata:
+        labels:
+          app: thanos-store
+      spec:
+        containers:
+        - args:
+          - store
+          - --data-dir=/var/thanos/store
+          - --grpc-address=0.0.0.0:10901
+          - --objstore.config=$(OBJSTORE_CONFIG)
+          env:
+          - name: OBJSTORE_CONFIG
+            valueFrom:
+              secretKeyRef:
+                key: thanos.yaml
+                name: ${THANOS_CONFIG_SECRET}
+          - name: AWS_ACCESS_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                key: aws_access_key_id
+                name: ${THANOS_S3_SECRET}
+          - name: AWS_SECRET_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                key: aws_secret_access_key
+                name: ${THANOS_S3_SECRET}
+          image: ${IMAGE}
+          name: thanos-store
+          ports:
+          - containerPort: 10901
+            name: grpc
+          volumeMounts:
+          - mountPath: /var/thanos/store
+            name: data
+            readOnly: false
+        volumes:
+        - emptyDir: {}
+          name: data
+    volumeClaimTemplates: []
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      service.alpha.openshift.io/serving-cert-secret-name: telemeter-server-shared
+    labels:
+      k8s-app: telemeter-server
+    name: telemeter-server
+    namespace: ${NAMESPACE}
+  spec:
+    clusterIP: None
+    ports:
+    - name: external
+      port: 8443
+      targetPort: external
+    - name: internal
+      port: 8081
+      targetPort: internal
+    - name: cluster
+      port: 8082
+      targetPort: cluster
+    selector:
+      k8s-app: telemeter-server
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: telemeter-server
+    namespace: ${NAMESPACE}
+- apiVersion: apps/v1beta2
+  kind: StatefulSet
+  metadata:
+    name: telemeter-server
+    namespace: ${NAMESPACE}
+  spec:
+    podManagementPolicy: Parallel
+    replicas: 3
```

Comment:
Should we parameterize this? Telemeter in production needs to be using 10 replicas, for example.

Reply:
This was not parameterized in the current Telemeter version. We should definitely do that. Can we do that in a follow-up PR? :)

https://github.com/openshift/telemeter/blob/25f46281db14a545d07b1b9fcca7105b4e27c629/manifests/server/list.yaml#L101